services/payments/scripts/fake_payment_gateway.py

```diff
@@ -166,22 +328,40 @@ def pay_with_payment_method(
     return router
 
 
+@asynccontextmanager
+async def _app_lifespan(app: FastAPI):
+    state_path = Path("app.state.payments.ignore.json")
+    if state_path.exists():
+        app.state.payments = parse_file_as(dict[str, Any], state_path)
+
+    yield
+
+    state_path.write_text(json.dumps(jsonable_encoder(app.state.payments), indent=1))
+
+
 def create_app():
     app = FastAPI(
         title="fake-payment-gateway",
         version="0.2.0",
-        servers=[
-            {
-                "url": "{scheme}://{host}:{port}",
-                "description": "development server",
-                "variables": {
-                    "scheme": {"default": "http"},
-                    "host": {"default": "localhost"},
-                    "port": {"default": "8080"},
-                },
-            }
-        ],
+        # servers=[
```

Comment:
Do you want to leave this commented code here?

Reply:
nope! thx